### PR TITLE
Clean up snapshot tests some

### PR DESCRIPTION
**Pull request by CAD97**
r? @Centril 
cc @eddyb https://github.com/rust-lang-nursery/wg-grammar/pull/30#discussion_r281025264

The snapshots should be a bit easier to read now. (I'll try to experiment with .gitattributes in my fork.)

### CODE REVIEW

File: src/bin/snapshots/snapshots__ModuleContents.hello-world.input.snap [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
+                            Stmt::Expr(
+                                Expr {
+                                    attrs: [],
+                                    kind: ExprKind::Path(
+                                        QPath::Unqualified(
+                                            Path {
+                                                path: RelativePath {
+                                                    segments: [
+                                                        PathSegment {
                                                             ident: _
```
Comment (Centril) [inline src/bin/snapshots/snapshots__ModuleContents.hello-world.input.snap:30]: Hmm; we are not showing `IDENT`s -- would be nice to fix in a different PR?

**Review by Centril — APPROVED** [review]

File: src/bin/snapshots/snapshots__ModuleContents.hello-world.input.snap (hunk 1 above)
Comment (CAD97) [reply src/bin/snapshots/snapshots__ModuleContents.hello-world.input.snap:30]: This is a gll issue (cc @eddyb): the debug representation is `ident: Span..Span` with the `proc_macro2` shim.

File: src/bin/snapshots/snapshots__ModuleContents.hello-world.input.snap [hunk 2]
```diff
@@ -36,43 +36,43 @@ Ok(
...
                                             } | ExprKind::Tuple {
-                                                attrs: _ => [],
-                                                exprs: _ => [
-                                                    _ => Expr {
-                                                        attrs: _ => [],
-                                                        kind: _ => ExprKind::Literal(_)
+                                                attrs: [],
+                                                exprs: [
+                                                    Expr {
+                                                        attrs: [],
```
Comment (eddyb) [inline src/bin/snapshots/snapshots__ModuleContents.hello-world.input.snap:55]: I wonder whether we should replace `\n *\w+: \[\],\n` with `\n`, to omit empty stuff like this.
